src/test/groovy/org/jenkinsci/gradle/plugins/jpi/JpiHplManifestSpec.groovy

```diff
@@ -15,8 +15,8 @@ class JpiHplManifestSpec extends Specification {
             apply plugin: 'jpi'
         }
         def libraries = [
-                new File(project.buildDir, 'classes/java/main'),
-                new File(project.buildDir, 'resources/main'),
+                new File(project.rootDir, 'src/main/resources'),
+                new File(project.buildDir, 'libs/test.jar'),
```

Comment:
I think the `classes` should remain as the libraries here. The built `jar` file shouldn't be there.

I think external dependencies need to be added here.

Reply:
This was actually broken already, but the test here  was not correct (afterEvaluate {} blocks were not executed and hence the build was not fully configured).

The test assertions are now back to what they were and the problem is fixed.